bin/hsdata

```diff
@@ -191,18 +203,22 @@ def process_ensemble_member(run_date, member, config):
                     forecast_data = make_forecast_track_data(model_tracks, run_date, member, config,
                                                              track_proc.model_grid.proj)
                     if patch_radius is not None:
-                        forecast_track_patches_to_netcdf(model_tracks, patch_radius, run_date, member, config)
+                        forecast_track_patches_to_netcdf(model_patches, patch_radius, run_date, member, config)
                     if config.json:
-                        forecast_tracks_to_json(model_tracks, run_date, member, config, track_proc.model_grid.proj)
+                        forecast_tracks_to_single_geojson(model_tracks, run_date, member,
+                                                          config, track_proc.model_grid.proj)
             elif len(model_tracks) > 0:
                 print(run_date, member, "Make Forecast Track Data")
                 forecast_data = make_forecast_track_data(model_tracks, run_date, member, config,
                                                          track_proc.model_grid.proj)
                 if patch_radius is not None:
                     print(run_date, member, "Track Data to netCDF")
-                    forecast_track_patches_to_netcdf(model_tracks, patch_radius, run_date, member, config)
+                    if model_patches is None:
+                        model_patches = model_tracks
+                    forecast_track_patches_to_netcdf(model_patches, patch_radius, run_date, member, config)
```

Comment:
The model patches do not seem to contain the model attributes which causes it to fail when converting to netCDF.